Neos.Neos/Configuration/Settings.yaml

```diff
@@ -560,6 +560,11 @@ Neos:
       applicationPackageKey: Neos.Neos
       applicationName: Neos
 
+    mvc:
+      routes:
+        'Neos.Neos':
+          position: 'start'
```

Comment:
And `position: 'start'` seems to be breaking, custom routes, wich should always be loaded before neos subroutes should still be loaded before neos.neos